vernac/declare.ml

```diff
@@ -114,7 +114,7 @@ let default_univ_entry = Entries.Monomorphic_entry Univ.ContextSet.empty
 (** [univsbody] are universe-constraints attached to the body-only,
    used in vio-delayed opaque constants and private poly universes *)
 let definition_entry_core ?(opaque=false) ?using ?(inline=false) ?feedback_id ?types
-    ?(univs=default_univ_entry) ?(eff=Evd.empty_side_effects) ?(univsbody=Univ.ContextSet.empty) body =
+    ?(univs=(default_univ_entry,UnivNames.empty_binders)) ?(eff=Evd.empty_side_effects) ?(univsbody=Univ.ContextSet.empty) body =
```

Comment:
Change default_univ_entry instead

Reply:
You mean some `default_named_univ_entry`? Done (hopefully as you intended it).

----
More generally, there is something a bit unaesthetic in this PR which I don't know how to solve. One would like to give a proper name to `Entries.universes_entry * UnivNames.universe_binders` but I don't know what to choose. Morally, this is like `Entries.universes_entry` but we cannot modify the meaning `Entries.universes_entry` since the kernel does not know binders.

I tried in an extra commit to add `named_universes_entry` in `uState.ml`? Tell me if you prefer it or no.

Reply:
>You mean some default_named_univ_entry? Done (hopefully as you intended it).

No, just add the binders to default_univ_entry.

Reply:
There is another `default_univ_entry` later in the file without binders, so I cannot just change its type.